tests/sles4sap/saptune/mr_test.pm

```diff
@@ -300,7 +298,7 @@ sub run {
     } elsif ($test =~ m/^(x86_64|ppc64le)$/) {
         $self->test_x86_64  if (check_var('BACKEND', 'ipmi'));
         $self->test_ppc64le if (get_var('OFW'));
-        $self->test_bsc1152598;
+        #$self->test_bsc1152598;
```

Comment:
Please just add a comment to tell exactly why we have commented code here.

Reply:
Done.